lib/Auth/Process/IsCesnetEligible.php

```diff
@@ -104,32 +126,53 @@ public function process(&$request)
+                    $this->cesnetEligibleLastSeenValue =
+                        ($this->ldapConnector->getUserAttributes($user, [$this->ldapAttrName]))['isCesnetEligible'][0];
+                } else {
+                    $this->cesnetEligibleLastSeenAttribute = $this->rpcConnector->get(
+                        'attributesManager',
+                        'getAttribute',
+                        ['user' => $user->getId(), 'attributeName' => $this->rpcAttrName,]
+                    );
+                    $this->cesnetEligibleLastSeenValue = $this->cesnetEligibleLastSeenAttribute['value'];
+                }
 
-            if ((!empty($this->eduPersonScopedAffiliation) && $this->isCesnetEligible())
-                || $isHostelVerified
-            ) {
-                $this->cesnetEligibleLastSeen['value'] = date("Y-m-d H:i:s");
+                if ((!empty($this->eduPersonScopedAffiliation) && $this->isCesnetEligible()) || $isHostelVerified) {
+                    $oldCesnetEligibleLastSeenValue = $this->cesnetEligibleLastSeenValue;
+                    $this->cesnetEligibleLastSeenValue = date("Y-m-d H:i:s");
+
+                    if ($this->cesnetEligibleLastSeenAttribute === null) {
+                        $this->cesnetEligibleLastSeenAttribute = $this->rpcConnector->get(
+                            'attributesManager',
+                            'getAttribute',
+                            ['user' => $user->getId(), 'attributeName' => $this->rpcAttrName,]
+                        );
+                    }
+                    $this->cesnetEligibleLastSeenAttribute['value'] = $this->cesnetEligibleLastSeenValue;
 
-                if (!empty($user)) {
                     $this->rpcConnector->post(
                         'attributesManager',
                         'setAttribute',
-                        array('user' => $user->getId(), 'attribute' => $this->cesnetEligibleLastSeen,)
+                        array('user' => $user->getId(), 'attribute' => $this->cesnetEligibleLastSeenAttribute,)
+                    );
+
+                    Logger::debug(
+                        "cesnet:IsCesnetEligible - Value of attribute isCesnetEligibleLastSeen was changed from " .
+                        $oldCesnetEligibleLastSeenValue . " to " . $this->cesnetEligibleLastSeenValue
                     );
                 }
             }
         } catch (Exception $ex) {
             Logger::warning("cesnet:IsCesnetEligible - " . $ex->getMessage());
         }
 
-        if ($this->cesnetEligibleLastSeen['value'] != null) {
-            $request['Attributes'][$this->attrName] = array($this->cesnetEligibleLastSeen['value']);
+        if ($this->cesnetEligibleLastSeenValue != null) {
+            $request['Attributes'][$this->returnAttrName] = array($this->cesnetEligibleLastSeenValue);
+            Logger::debug(
+                "cesnet:IsCesnetEligible - Attribute " . $this->returnAttrName . " was sent with value " .
```

Comment:
There will be better use msg that attribute was setted to value.

Reply:
Done